scout/adapter/mongo/variant.py

```diff
@@ -516,14 +527,14 @@ def institute_causatives(self, institute_obj, limit_genes=None):
         )
         causative_ids = set()
         for case_event in causative_events:
-            case_obj = self.case(case_event.get("case"))
+            case_obj = self.case(case_event.get("case"), projection=CASE_CAUSATIVES_PROJECTION)
             if case_obj is None:
                 continue
 
             for var_id in case_obj.get("causatives", []):
                 causative_ids.add(var_id)
 
-            for var_id, _ in case_obj.get("patial_causatives", {}).items():
```

Comment:
Whoops, was this a bug?

Reply:
Mmm, at least I haven't seen `patial_causatives` elsewhere! 😆 Its in the list, so presumably not much harm done.